daemon/api_test.go

```diff
@@ -1293,11 +1293,67 @@ func (s *apiSuite) TestInstallLicensedIntegration(c *check.C) {
 
 // Tests for GET /2.0/interfaces
 
+func makeTypicalPlug(ifaceName string) *interfaces.Plug {
```

Comment:
I suggest `s/Typical//` over this file. It's a lot of typicalness which doesn't really improve the readability much. Out of its name we know this is a local and private function in a test file.